To: vim_dev@googlegroups.com
Subject: Patch 7.3.364
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.364 (after 7.3.353)
Problem:    Can't compile on HP-UX. (John Marriott)
Solution:   Only use TTYM_URXVT when it is defined.
Files:	    src/term.c


*** ../vim-7.3.363/src/term.c	2011-11-30 14:47:12.000000000 +0100
--- src/term.c	2011-12-01 20:53:57.000000000 +0100
***************
*** 4060,4066 ****
  		    if (tp[1 + (tp[0] != CSI)] == '>' && j == 2)
  		    {
  			/* if xterm version >= 95 use mouse dragging */
! 			if (extra >= 95 && ttym_flags != TTYM_URXVT)
  			    set_option_value((char_u *)"ttym", 0L,
  						       (char_u *)"xterm2", 0);
  			/* if xterm version >= 141 try to get termcap codes */
--- 4060,4070 ----
  		    if (tp[1 + (tp[0] != CSI)] == '>' && j == 2)
  		    {
  			/* if xterm version >= 95 use mouse dragging */
! 			if (extra >= 95
! # ifdef TTYM_URXVT
! 				&& ttym_flags != TTYM_URXVT
! # endif
! 				)
  			    set_option_value((char_u *)"ttym", 0L,
  						       (char_u *)"xterm2", 0);
  			/* if xterm version >= 141 try to get termcap codes */
*** ../vim-7.3.363/src/version.c	2011-11-30 17:20:18.000000000 +0100
--- src/version.c	2011-12-01 20:56:55.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     364,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
224. You set up your own Web page. You set up a Web page for each
     of your kids... and your pets.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
